### ISSUE TITLE
Screenshots of premium functions

### ISSUE THREAD

**Issue by lifehaschanged** [decaf-dev/obsidian-vault-explorer#125]
I'd be glad if you could provide us some screenshots of the premium functions
in your plugin.
That would be important for users who wants to buy a license of your plugin.
Thanks in advance!

**Comment by decaf-dev** [comment]
Sure thing! I have been bogged down with development, so I will transition to the website soon.

**Comment by decaf-dev** [comment]
Yes. I will add PayPal today. Thanks for the feedback. I appreciate it!

**Comment by decaf-dev** [comment]
@lifehaschanged It has been sent. Please DM on Kofi if you have any issues :)
